docs/install/ha.md

```diff
@@ -0,0 +1,88 @@
+
+Setting up an HA cluster requires the following steps:
+
+1. Configure a fixed registration address
+1. Launch the first server node
+1. Join additional server nodes
+1. Join agent nodes
+
+### 1. Configure the Fixed Registration Address
+
+Server nodes beyond the first one and all agent nodes need a URL to register against. This can be the IP or hostname of any of the server nodes, but in many cases those may change over time as nodes are created and destroyed. Therefore, you should have a stable endpoint in front of the server nodes.
+
+This endpoint can be set up using any number approaches, such as:
+
+* A layer-4 (TCP) load balancer
+* Round-robin DNS
+* Virtual or elastic IP addresses
+
+This endpoint can also be used for accessing the Kubernetes API. So you can, for example, modify your [kubeconfig](https://kubernetes.io/docs/concepts/configuration/organize-cluster-access-kubeconfig/) file to point to it instead of a specific node.
+
+Note that the `rke2 server` process listens on port `9345` for new nodes to register. The Kubernetes API is served on port `6443`, as normal. Configure your load balancer accordingly.
+
+### 2. Launch the first server node
+The first server node establishes the secret that other server or agent nodes will register with when connecting to the cluster.
+
+To specify your own pre-shared secret as the token, set the `token` argument on startup.
+
+If you do not specify a pre-shared secret, RKE2 will generate one and place it at `/var/lib/rancher/rke2/server/node-token`.
+
+To avoid certificate errors with the fixed registration address, you should launch the server with the `tls-san` parameter set. This option adds an additional hostname or IP as a Subject Alternative Name in the server's TLS cert, and it can be specified as a list if you would like to access via both the IP and the hostname.
+
+Here is an example of what the RKE2 config file would look like if you are following this guide:
+```
+token: my-shared-secret
+tls-san:
+  - my-kubernetes-domain.com
+  - another-kubernetes-domain.com
+```
+
+#### 2a. Optional: Consider server node taints
+By default, server nodes will be schedulable and thus your workloads can get launched on them. If you wish to have a dedicated control plane where no user workloads will run, you can use taints. The `node-taint` parameter will allow you to configure nodes with taints. Here is an example of adding a node taint to the configuration file:
+```terminal
+node-taint
+  - "CriticalAddonsOnly=true:NoExecute"
+```
+
+### 3. Launch additional server nodes
+Additional server nodes are launched much like the first, except that you must specify the `server` and `token` parameters so that they can successfully connect to the initial server node.
+
+Here is an example of what the RKE2 config file (at `/var/lib/rancher/rke2/config.yaml`) would look like for addtional server nodes if you are following this guide:
```

Comment:
Should the location be /etc/rancher/rke2/config.yaml

Reply:
Yep. Thanks!!

Reply:
additional typo